src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -163,84 +168,156 @@ private void initializeRamseteCommands() {
               new ShooterPressed(
                   shooterSubsystem,
                   limelightSubsystem,
                   cdsSubsystem,
-                  false)); // shoot the two acquired balls
+                  false), // shoot the two acquired balls
+              threeBallParallel2, // grab last ball
+              threeBallRamseteCommands[3], // come back to shoot
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false));
 
-      // fourBallCommand
-      // fiveBallCommand
+      // --------------------------------------------
 
+      ParallelDeadlineGroup fourBallParallel1 =
+          new ParallelDeadlineGroup(
+              fourBallRamseteCommands[0],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      ParallelDeadlineGroup fourBallParallel2 =
+          new ParallelDeadlineGroup(
+              fourBallRamseteCommands[2],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      // similar path to threeball, now just getting the additional ball at terminal
+      fourBallCommand =
+          new SequentialCommandGroup(
+              new WaitCommand(initialWaitTime),
+              fourBallParallel1,
+              fourBallRamseteCommands[1],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fourBallParallel2,
+              fourBallRamseteCommands[3],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false));
+
+      // ---------------------------------------
+
+      ParallelDeadlineGroup fiveBallParallel1 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[0],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      ParallelDeadlineGroup fiveBallParallel2 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[2],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      ParallelDeadlineGroup fiveBallParallel3 =
+          new ParallelDeadlineGroup(
+              fiveBallRamseteCommands[4],
+              new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
+
+      fiveBallCommand =
+          new SequentialCommandGroup(
+              new WaitCommand(initialWaitTime),
+              fiveBallParallel1,
+              fiveBallRamseteCommands[1],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fiveBallParallel2,
+              fiveBallRamseteCommands[3],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
+              fiveBallParallel3,
+              fiveBallRamseteCommands[5],
+              new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false));
```

Comment:
It seems like each path has a similar structure. 

1. wait
2. intake deadline path
3. return to fender path
4. shoot
5. goto 2

A good future update would be to figure out a iterative (or recursive) way to generate these so you don't have to copy paste so much